Resources/Prototypes/Voice/disease_emotes.yml

```
@@ -21,27 +21,27 @@
 
 - type: emote
   id: MonkeyScreeches
   category: Vocal
-  chatMessages: [ визжит ]
+  chatMessages: [ chat-emote-msg-monkeyscreeches ]
 
 - type: emote
   id: RobotBeep
   category: Vocal
-  chatMessages: [ бипает ]
+  chatMessages: [ chat-emote-msg-robotbeep ]
```

yamllint

[error] too many spaces inside brackets

(brackets)